### ISSUE TITLE
Page Selection GUI

### ISSUE THREAD

**Issue by blackm61** [Isaiahensley/Aquatic-RIG#52]
This issue is for our website's GUI.

**Comment by blackm61** [comment]
![image](https://github.com/Isaiahensley/Aquatic-RIG/assets/143751284/5d5b01a6-ae54-422a-9a85-23ec9ae6ee8c)


![image](https://github.com/Isaiahensley/Aquatic-RIG/assets/143751284/0e143cae-312c-4732-a287-0914bc740a8a)


Considering changing the select page selectbox, to just buttons instead of the selectbox (like in the second picture)

**Comment by spaude11** [comment]
All the best with this part. Let us know if there's any assistance you needs from us.

**Comment by blackm61** [comment]
![image](https://github.com/Isaiahensley/Aquatic-RIG/assets/143751284/e0901a39-c205-4065-9861-37b877d2e4ca)

![image](https://github.com/Isaiahensley/Aquatic-RIG/assets/143751284/8b0325eb-052c-4f5e-939d-e357d557dab1)

**Comment by blackm61** [comment]
Changed the side bar to have buttons instead of the drop down menu. It also removes the currently selected page when clicked. But I think it looks better with the drop down menu instead. Will discuss with team about what's best.

**Comment by spaude11** [comment]
To enhance the clarity and productivity of our standups, I require that all issues presented adhere to a detailed format, as exemplified below. This approach will not only improve our understanding of each issue but also streamline our collaborative efforts to address them.

Sample Format for Issue Presentation:

Description: Clearly state the problem, including any specific error messages or unexpected behavior. Detail the context in which the issue occurs.

Steps to Reproduce: Provide a step-by-step guide on how to encounter the issue, making it easier for team members to replicate and understand the problem.

Expected vs. Actual Behavior: Describe what the expected behavior should be and how the actual outcome deviates from this expectation.

Code Snippets: Include relevant code snippets or configurations that relate to the issue, aiding in a more technical understanding and discussion.

Environment Details: Mention the operating system, software versions, and any other environment specifics that could influence the issue.

For Issue Resolution:

Detail the investigation, proposed changes, and the outcome, ensuring to describe how the resolution addresses the issue. Include modified code snippets and a summary of the impact of these changes.

Struture your issue according to this. If you don't encounter any errors then your goal can be the main focus of your issue discription

**Comment by blackm61** [comment]
**Description:** 
Currently, the page selection GUI has a drop down selection box. The page selection box is featured across all pages. Streamlit is limiting with the options that we have, so it's difficult to find something that works and is clean for our page selection. I've discussed with Isaiah changing it to have buttons instead of the select box.  And I changed some of the code to make it work this way. 

I had to use 'st.rerun' to refresh parts of the page, so that the page a user has currently selected would not show up in the sidebar.  There are many limitations and issues that can happen as a result of using st.rerun, so I don't think that this is the best way to go foward. We could still have buttons, but I think the overall website looks best with the original drop down page selection box. 

**Steps to Reproduce:** 
I've provide some code at the bottom of this comment to replicate the buttons. You add the snippets of code to aquaticrig_gui.py and then run the code to the local host. 

Expected vs. Actual Behavior: 
The code is supposed to create a layout for our GUI that allows you to select pages with the buttons. This works as expected, however I had to use 'st.rerun' to refresh parts of the page, so that the page a user has currently selected would not show up in the sidebar.  There are many limitations and issues that can happen as a result of using st.rerun. Documentations says that some caveats to st.rerun are: _Additional script runs may be inefficient and slower, Excessive reruns may complicate your app's logic and be harder to follow, If misused, infinite looping may crash your app._

Code Snippets: 
**Environment Details:** 
Pycharm (python 3)
localhost streamlit sharing website

For Issue Resolution:
